app/views/dashboards/show.html.erb

```diff
@@ -40,14 +40,14 @@
       <li>
         <%= link_to [:measurements] do %>
           <em>
-            <%= number_with_delimiter current_user.measurements.count %>
+            <%= number_to_human current_user.measurements.approximate_count %>
```

Comment:
Is `approximate_count` per-user or total?

Reply:
Total is the first priority.Per-user can be added later.

Reply:
So we shouldn’t change it here because it will show the same count for all users

Reply:
True. We cannot use this trick...